coding-the-legislation/37_inferences.md

```diff
@@ -0,0 +1,26 @@
+# Inferences
```

Comment:
I must say I don't really know where to either find or write things in the current information architecture. If the current process for finding the place is “@cbenz knows”, then let's keep with that process for the moment  😄